packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -100,8 +100,11 @@ class FlutterDevice {
     if (flutterViews == null || flutterViews.isEmpty)
       return;
     for (FlutterView view in flutterViews) {
-      if (view != null && view.uiIsolate != null)
+      if (view != null && view.uiIsolate != null) {
+        // Manage waits specifically below.
```

Comment:
@johnmccutchan maybe you're best for our understanding. Was the original intent here not to have some rpc completion confirmation but to wait a predefined amount of time?